package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bluejay/schema",
-  "version": "1.9.5",
+  "version": "1.10.0",
```

Comment:
That would be 2.0.0 now. Types are including in the package and we're breaking them.

Reply:
Whoops, that must have been from some ancient local changes I had on my computer.

Yeah, definitely 2.0.0 👍